osmosis-1/osmosis-1.assetlist.json

```diff
@@ -374,5 +374,33 @@
         "svg": "https://raw.githubusercontent.com/osmosis-labs/assetlists/main/images/juno.svg"
       }
     }
+    {
+      "description": "The native token of IXO Chain",
+      "denom_units": [
+        {
+          "denom": "ibc/F3FF7A84A73B62921538642F9797C423D2B4C4ACB3C7FCFFCE7F12AA69909C4B",
+          "exponent": 0,
+          "aliases": [
+            "uixo"
+          ]
+        },
+        {
+          "denom": "IXO",
```

Comment:
I'm going to recommend making this denom be lowercase, as that seems to be the standard for some reason

Reply:
```suggestion
          "denom": "ixo",
```